deploy/kubernetes/setup.sh

```diff
@@ -58,18 +58,38 @@ do
  in
  c) COLLECTOR_NAME=${OPTARG};;
  k) CLUSTER_NAME=${OPTARG};;
+ n) NAMESPACE=${OPTARG};;
  esac
 done
 shift "$(($OPTIND -1))"
 
 TIME=`timestamp`;
 
 if [ -z $COLLECTOR_NAME ]; then
-  COLLECTOR_NAME="kubernetes-$TIME";
+  if [ -z $SUMO_COLLECTOR_NAME ]; 
+  then
+    COLLECTOR_NAME="kubernetes-$TIME";
+  else
+    COLLECTOR_NAME=$SUMO_COLLECTOR_NAME;
+  fi
 fi
 
 if [ -z $CLUSTER_NAME ]; then
-  CLUSTER_NAME="kubernetes-$TIME";
+  if [ -z $KUBERNETES_CLUSTER_NAME ]
+  then
+    CLUSTER_NAME="kubernetes-$TIME";
+  else
+    CLUSTER_NAME=$KUBERNETES_CLUSTER_NAME
+  fi
+fi
+
+if [ -z $NAMESPACE]; then
+  if [ -z $SUMO_NAMESPACE ]
+  then
+    NAMESPACE="sumo"
```

Comment:
Can we make the default `sumologic` to stay consistent with the original default value?

Reply:
Good call - Didn't intend to change the default.